controller/adoptopenebs/apiserver.go

```diff
@@ -100,7 +101,7 @@ func (p *Planner) formMayaAPIServerConfig(mayaAPIServer *unstructured.Unstructur
 		if err != nil {
 			return err
 		}
-		if containerName == "maya-apiserver" {
+		if containerName == "maya-apiserver" || strings.Contains(containerName, "apiserver") {
```

Comment:
Will it cause an issue here if there is another container named mayastor-apiserver in future?

Reply:
This container is specific to maya-apiserver @avishnu and this seems to be the only way to identify this container as of now.